activiti-engine/src/test/java/org/activiti/engine/test/bpmn/event/error/BoundaryErrorEventTest.java

```diff
@@ -54,14 +54,26 @@ public void testThrowErrorWithEmptyErrorCode() {
     }
   }
 
-  @Deployment
+  public void testThrowErrorWithoutErrorCode() {
+    try {
+      repositoryService.createDeployment().addClasspathResource("org/activiti/engine/test/bpmn/event/error/BoundaryErrorEventTest.testThrowErrorWithoutErrorCode.bpmn20.xml").deploy();
+      fail("ActivitiException expected");
+    } catch (ActivitiException re) {
+    }
+  }
+
   public void testCatchErrorOnEmbeddedSubprocessWithEmptyErrorCode() {
+    try {
     testCatchErrorOnEmbeddedSubprocess();
+    } catch (ActivitiException re) {
+    }
   }
 
-  @Deployment
   public void testCatchErrorOnEmbeddedSubprocessWithoutErrorCode() {
+    try {
     testCatchErrorOnEmbeddedSubprocess();
+    } catch (ActivitiException re) {
```

Comment:
If it's suppose to throw an exception you need to call `fail` method or migrate to `catchThrowable`.